src/quo/components/links/internal_link_card/user/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo.components.links.internal-link-card.user.view
+  (:require
+    [quo.components.icon :as icon]
+    [quo.components.links.internal-link-card.user.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [react-native.linear-gradient :as linear-gradient]))
+
+(defn- subtitle-comp
+  [subtitle emojis]
+  [rn/view
+   [text/text
+    {:size                :paragraph-2
+     :number-of-lines     2
+     :accessibility-label :subtitle
+     :style               {:margin-bottom 12}} subtitle]
+   [rn/view {:flex-direction :row}
+    (doall
+     (map-indexed (fn [index emoji]
+                    ^{:key (str emoji "-" index)}
```

Comment:
Correct me if I'm wrong, but with production data, there's never a moment where the user's sequence of emojis change that would cause React to re-render (this is a very static piece of data), so to silence React from complaining about non-unique keys, just `index` suffices.

> We don’t recommend using indexes for keys if the order of items may change. 
>
> -- https://legacy.reactjs.org/docs/lists-and-keys.html#keys